_posts/2024-04-02-shared-conda-tutorial.md

```diff
@@ -0,0 +1,103 @@
+
+On multi-user systems (e.g. shared Linux servers), multiple users can share  the same conda installation and the environments thererin. This is a good idea and usually works well when only already existing environments are re-used, but can become tricky when users also want to create environments (using `conda create`) into the shared conda installation. On a standard Linux system, this often results in permission problems when (re-)downloading packages and overwriting files that were created by others.
+
+Here we describe a setup with a shared (read-only) conda installation, with re-usable shared environments, and private per-user environments.
+
+## Tutorial
+
+### Creating the shared conda installation
+
+First, create a shared conda installation folder that is read-only for all regular users. Shared conda environments therein, which are available to all users, must be created by the sysadmin (or with a user account with the correct privileges for the conda folder). 
+
+In this tutorial we'll use `/apps/conda` as the path, but please choose something appropriate for your system.
+
+We install conda by downloading and running the [Miniconda installer](https://repo.anaconda.com/miniconda/), but feel free to use the [full Anaconda installer](https://repo.anaconda.com/archive/) if that is your preference. We also set up folder permissions and an initial conda config, which includes the recommended [bioconda](https://bioconda.github.io/) and [conda-forge](https://conda-forge.org/) channels.
+
+```bash
+# Create shared conda folder
+CONDA_PATH="/apps/conda"
+sudo mkdir -pv "${CONDA_PATH}"
+sudo chmod 755 "${CONDA_PATH}"
+
+# Download Miniconda installer
+CONDA_URL="https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh"
+wget "${CONDA_URL}" --output-document=/tmp/conda_installer.sh
+
+# Install Miniconda
+# -b          run install in batch mode (without manual intervention),
+#             it is expected the license terms (if any) are agreed upon
+# -u         update an existing installation (needed to install in existing dir)
+# -p PREFIX  install prefix, defaults to $PREFIX, must not contain spaces.
+sudo bash /tmp/conda_installer.sh -b -u -p "${CONDA_PATH}"
+
+# Set default conda settings for all users
+# (these will show up in $CONDA_PATH/.condarc)
+sudo "${CONDA_PATH}"/bin/conda config --system --set channel_priority strict
+sudo "${CONDA_PATH}"/bin/conda config --system --add channels defaults
+sudo "${CONDA_PATH}"/bin/conda config --system --add channels bioconda
+sudo "${CONDA_PATH}"/bin/conda config --system --add channels conda-forge
+```
+
+Users can override default conda settings in their own `$HOME/.condarc` configuration files.
+
+
+### Creating environments in the shared conda installation
+Any user with sudo privileges can create environments in the shared folder:
+```bash
+# activate base 
```

Comment:
I would probably recommend paring this down to a simple: 
```
sudo /apps/conda/bin/conda create -n myenv ...
```
This will create `myenv` in the shared environments folder (i.e. `/apps/conda/envs/myenv`) so it will be accessible for all users.